contracts/adapters/Managing.sol

```diff
@@ -98,6 +105,12 @@ contract ManagingContract is IManaging, DaoConstants, MemberGuard {
         require(_proposalId < type(uint64).max, "proposalId too big");
         uint64 proposalId = uint64(_proposalId);
         ProposalDetails memory proposal = proposals[_proposalId];
+				bytes32[] memory keys = proposal.keys;
+				uint256[] memory values = proposal.values;
+				require(
```

Comment:
do we really need to do this test again? If we have tested that when we create the entries we don't need to do it again when we process it no ?

Reply:
That should be the case, yes.